package.json

```diff
@@ -48,7 +51,7 @@
     "lint": "npm run check-versions && eslint --ext .js,.jsx --ignore-path .gitignore .",
     "lint:flow": "babel-node scripts/flow-check.js",
     "plop": "plop",
-    "bootstrap": "yarn && npm run check-versions && lerna bootstrap",
+    "bootstrap": "yarn && npm run check-versions && lerna run prepublish",
```

Comment:
Do we want to run prepublish for bootstrapping? Perhaps this is a problem just with my workflow but I'm always watching the repo to do changes so don't need to compile files as part of bootstrap. Or does prepublish do more than just compile now?

Though probably it makes sense as the default. With workspaces I can just run `yarn` now to install everything everywhere right?

Reply:
It's safe thing to do, and required for tests to pass. The issue is on packages that depend on each other, if `gatsby-foo` needs `gatsby-foo` jest isn't going to resolve `gatsby-foo` to the `src` directory so the compiled files need to exist. It's not strictly required to get going tho.  You can just run `yarn` and start working and everything will be installed 👍